packages/schematics/angular/migrations/update-8/update-dependencies.ts

```diff
@@ -11,55 +11,29 @@ import { latestVersions } from '../../utility/latest-versions';
 
 export function updateDependencies() {
   return (host: Tree) => {
-    let current = getPackageJsonDependency(host, '@angular-devkit/build-angular');
-    if (current && current.version !== latestVersions.DevkitBuildAngular) {
-      addPackageJsonDependency(
-        host,
-        {
-          type: current.type,
-          name: '@angular-devkit/build-angular',
-          version: latestVersions.DevkitBuildAngular,
-          overwrite: true,
-        },
-      );
-    }
 
-    current = getPackageJsonDependency(host, '@angular-devkit/build-ng-packagr');
-    if (current && current.version !== latestVersions.DevkitBuildNgPackagr) {
-      addPackageJsonDependency(
-        host,
-        {
-          type: current.type,
-          name: '@angular-devkit/build-ng-packagr',
-          version: latestVersions.DevkitBuildNgPackagr,
-          overwrite: true,
-        },
-      );
-    }
+    const dependenciesToUpdate: Record<string, string> = {
+      '@angular-devkit/build-angular': latestVersions.DevkitBuildAngular,
+      '@angular-devkit/build-ng-packagr': latestVersions.DevkitBuildNgPackagr,
```

Comment:
can you also add `'@angular-devkit/build-webpack'` as well?

Reply:
Added